resources/views/users/DispUsers.blade.php

```diff
@@ -3,15 +3,15 @@
 <div class='container'>
```

Comment:
In this file please change file name. Must use camelCase format for blade name not StudlyCase format